doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -612,6 +612,12 @@ julia> 2(x-1)^2 - 3(x-1) + 1
 3
 ```
 
+!!! warning
+    Notice that, in this case, the precedence of numeric literal coefficient is
+    still the same as that of the unary operators. Hence, the expression
+    `6/2(2+1)` leads to `1` instead of the conventional result `9`, which is
```

Comment:
Let's keep this notation but change it from a `warning` which sounds a bit too dire to a `note`.